src/ca/mcgill/cs/jetuml/diagram/Diagram.java

```diff
@@ -130,6 +130,16 @@ else if(pTest instanceof ParentNode)
 		return false;
 	}
 
+	/**
+	 * @param pNode
+	 * @return True if pNode is a root node of the Diagram
+	 */
+	public boolean rootNodesContain(Node pNode) 

```

Comment:
Suggested renaming: `containsAsRoot(Node pNode)`

Reply:
Thanks! I renamed it.